pkg/storaged/client.js

```diff
@@ -499,12 +564,18 @@ function init_model(callback) {
             function() {
                 client.manager_lvm2 = proxy("Manager.LVM2", "Manager");
                 client.manager_iscsi = proxy("Manager.ISCSI.Initiator", "Manager");
-                return Promise.allSettled([client.manager_lvm2.wait(), client.manager_iscsi.wait()])
-                        .then(() => {
-                            client.features.lvm2 = client.manager_lvm2.valid;
-                            client.features.iscsi = (client.manager_iscsi.valid &&
-                                                            client.manager_iscsi.SessionsSupported !== false);
-                        });
+                client.manager_btrfs = proxy("Manager.BTRFS", "Manager");
+                return Promise.allSettled([client.manager_lvm2.wait(),
+                                           client.manager_iscsi.wait(),
+                                           client.manager_btrfs.wait()])
+                    .then(() => {
+                        client.features.lvm2 = client.manager_lvm2.valid;
+                        client.features.iscsi = (client.manager_iscsi.valid &&
+                                                 client.manager_iscsi.SessionsSupported !== false);
+                        client.features.btrfs = client.manager_btrfs.valid;
+                        if (client.features.btrfs)
+                            btrfs_start_polling();
+                    });
```

Comment:
So this polling is required to watch subvolumes? For example when someone makes one outside of Cockpit, is this something we normally rely on udisks for?